jina/optimizers/__init__.py

```diff
@@ -155,6 +155,13 @@ def optimize_flow(
         return result_processor(study)
 
 
+def run_optimizer(args):
```

Comment:
Just a reminder to add better type hinting in this module

Reply:
thanks, done